ci-operator/step-registry/openshift/e2e/openstack/perfscale/conf-securitygroup/deprovision/openshift-e2e-openstack-perfscale-conf-securitygroup-deprovision-commands.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+# Deleting the network-perf security group
+NETWORK_PERF_SG=${NETWORK_PERF_SG:-"network-perf-sg"}
+
+if ! openstack security group show "${NETWORK_PERF_SG}" >/dev/null; then
+	echo "ERROR: The network-perf security group does not exist: ${NETWORK_PERF_SG}. Nothing to delete"
+    exit 0
+fi
+
+echo "Deleting the Security Group $NETWORK_PERF_SG"
+openstack security group delete $NETWORK_PERF_SG
```

Comment:
This could be a step that deletes all security groups that we created in the provision phase.

For example:
```bash
if [[ -d "${SHARED_DIR}/securitygroups" ]]; then
	for sg_id in "${SHARED_DIR}"/securitygroups/*; do
		echo "Deleting security group '$(basename $sg_id)'..."
		openstack security group delete "$(<"$sg_id")"
	done
fi
```